### PR TITLE
 Add hide! utility function. 

### PR DESCRIPTION
**Pull request by tkoolen**
On top of #96.

### CODE REVIEW

File: src/abstract_visualizer.jl [hunk 1]
```diff
@@ -8,6 +8,7 @@ function setprop! end
 function setcontrol! end
 
 
-# Convenient shortcuts for creating new objects
+# Convenient shortcuts
 setobject!(vis::AbstractVisualizer, geom::GeometryLike) = setobject!(vis, Object(geom))
 setobject!(vis::AbstractVisualizer, geom::GeometryLike, material::AbstractMaterial) = setobject!(vis, Object(geom, material))
+hide!(vis::AbstractVisualizer) = settransform!(vis, LinearMap(UniformScaling(false)))
```
Comment (rdeits) [inline src/abstract_visualizer.jl]: I think `setprop!(vis, "visible", false)` might be a more idiomatic way to do this. What do you think?

Comment (tkoolen) [reply src/abstract_visualizer.jl]: Yes. I thought you might have a suggestion like that, but I didn't know the right API call. I'll change it.

**Comment by tkoolen** [comment]
Done, but I guess this should now be `setvisible!(vis, visible::Bool)`.

**Comment by tkoolen** [comment]
Done.

**Comment by rdeits** [comment]
Thank you!

**Pull request merged**